app/models/manageiq/providers/azure/cloud_manager/vm.rb

```diff
@@ -1,7 +1,9 @@
 class ManageIQ::Providers::Azure::CloudManager::Vm < ManageIQ::Providers::CloudManager::Vm
+  include_concern 'Operations'
+
   def provider_object(connection = nil)
     connection ||= ext_management_system.connect
-    connection.instances[ems_ref]
+    connection
```

Comment:
Yeah it should return the `::Azure::Armrest::VirtualMachineService` object...what we are calling our "native" object.

Also, it seems weird to me to use .new...I would expect a .find, but that's not something for this PR.